__tests__/app.test.js

```diff
@@ -242,3 +242,28 @@ describe('POST /api/articles/:article_id/comments', () => {
       });
   });
 });
+
+describe('GET /api/users', () => {
+    test('200: returns an array of users objects ', () => {
+      return request(app)
+        .get('/api/users')
+        .expect(200)
+        .then(({ body }) => {
+            console.log(body);
+          expect(body.users).toHaveLength(4);
+          body.users.forEach((user) => {
+            expect(typeof user.avatar_url).toBe('string');
+            expect(typeof user.name).toBe('string');
+            expect(typeof user.username).toBe('string');
+          });
+        });
+    });
+    test('404: responds with an error message with invalid path', () => {
```

Comment:
We don't need this test here as it isnt specific to the `api/users` endpoint. Either remove it, or move it to a separate describe block just for incorrect endpoints 👍